src/components/QuizGame/index.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from "react";
+import { fetchQuizQuestions } from "../../components/QuizGameApi";
+import "antd/dist/antd.css";
```

Comment:
PTAL

Reply:
@majdajroudi  the value started with true because you will have to click on start button to start the game, so you will see empty page with a start button ,after starting the game will be displayed